src/main/java/com/epam/spring/homework3/service/impl/PlaceServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.spring.homework3.service.impl;
+
+import com.epam.spring.homework3.controller.dto.PlaceDTO;
+import com.epam.spring.homework3.service.PlaceService;
+import com.epam.spring.homework3.service.exception.NotFoundException;
+import com.epam.spring.homework3.service.mapper.PlaceMapper;
+import com.epam.spring.homework3.service.model.Place;
+import com.epam.spring.homework3.service.repository.PlaceRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
```

Comment:
Use @Service annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers